e2e/docker/docker.go

```diff
@@ -62,6 +63,11 @@ func (*Provider) Clean(ctx context.Context) error {
 
 // NewProvider returns a new Provider.
 func NewProvider(testnet types.Testnet, infd types.InfrastructureData, imgTag, graphQLURL string) *Provider {
+	platform := fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
+	if runtime.GOOS == "darwin" {
+		platform = fmt.Sprintf("linux/%s", runtime.GOARCH)
+	}
```

Comment:
can just do: `platform := fmt.Sprintf("linux/%s", runtime.GOARCH)`, no need for darwin if statement